packages/vaadin-lumo-styles/gulpfile.cjs

```diff
@@ -105,24 +99,6 @@ gulp.task('icons', (done) => {
 
         filenames.sort(sortIconFilesNormalized);
 
-        const ironIcons = `${createCopyright()}
```

Comment:
Nit: There's a leftover reference to iron-iconset-svg here: https://github.com/vaadin/web-components/pull/4879/files#diff-dbc63d5b8b55d2e1561f03daf1a665d2fee10aba6f3155f0a194033285b01eb6L99

Reply:
Thanks, fixed.